beyond-essence/inventory/toggle-switches.md

```diff
@@ -6,77 +6,63 @@ background_image: index.jpg
 
 ### Enhance cards with extensions
 
-The cockpit will be customizable.
+The cockpit is customizable.
 In order to allow merchants to adapt the cockpit to their needs, they can turn features on or off.
-This will be possible, for example, on single cards.
-
-**Example: Add the feature "product properties"**
-
-* If the feature is enabled, the merchant can add a list of product properties to use them e.g. in search filters.
-* If the feature is disabled, the merchant doesn't see this option, but can only use the description. For merchants with few and simple products this is sufficient, and the UI is less cluttered.
-
-The option is directly available.
-There is no need to confirm the action.
-
-{% image_custom image="/assets/img/pages/essence/switch-button_card_feature.png" width="50" align="center" %}
-
-💡 _Note: This use case is not yet implemented in the cockpit._
-
-### Other
+This is possible, for example, on single cards.
 
-Another example is the activation of additional variants for a product.
-If this feature is activated, the whole structure and workflow for this specific product will change.
+**Example: Enable pricing for variations**
 
-## Structure
+For the pricing of variation products, the merchant has two options:
 
-A toggle switch looks like a classical on/off switch.
-The two different states give a clear visual feedback so that the merchant is always aware if the toggle switch is turned on or off.
+* If the toggle is _disabled_ (default): The same price will be applied to all variations 
+* If the toggle is _enabled_: A table opens on the same card and allows the merchant to set an individual price for each variation.
 
-* A checkmark on a green background, toggled to the right, represents an active feature.
-* A cross on a beige background, toggled to the left, represents a deactivated feature.
+Both options don't have an impact on the functionality of the other cards, but enhance or reduce the functionality of the Pricing card only.
 
-On the right side of the toggle switch, a label clearly explains the action that can be taken.
-
-Depending on the complexity of the feature, additional content is shown below the activated toggle switch.
+### Switch between different modes
 
-{% image_custom image="/assets/img/pages/essence/switch-button-with-content.png" width="100" align="center" %}
+The third use case for toggle switches in our software is the activation/deactivation of different modes.
 
-For further information on the exact visualisation of addtional content, consult the design section.
+**Example: Enable delete mode**
 
-## Position
+On the Manage variations modal, the merchant can switch between two different modes:
 
-Toggle switches may be positioned on a card together with further UI elements that relate to a specific topic.
+* If the toggle is _disabled_ (default): The modal is in the Edit mode and allows the merchant to make changes to existing variation values.
```

Comment:
Maybe "is in the _Edit_ mode"?

Reply:
I don't use italics like this in the entire article, so I think it would be confusing to do so here.